src/python/pants/option/global_options.py

```diff
@@ -237,6 +237,12 @@ def register_bootstrap_options(cls, register):
     register('--enable-pantsd', advanced=True, type=bool, default=False,
              help='Enables use of the pants daemon (and implicitly, the v2 engine). (Beta)')
 
+    # Toggles v1/v2 `Task` vs `@rule` pipelines on/off.
+    register('--v1', advanced=True, type=bool, default=True,
+             help='Enables execution of v1 Tasks.')
+    register('--v2', advanced=True, type=bool, default=False,
```

Comment:
It took a little while to grok the split here... I think it makes sense though. Would probably want to explain the heck out of it in the help.